### PR TITLE
`floattype` does not exist .. changed to `float` which works with types

### PR DESCRIPTION
**Pull request by JeffreySarnoff**

### CODE REVIEW

File: src/ssim.jl [hunk 1]
```diff
@@ -86,7 +86,7 @@ function _ssim_map(iqi::SSIM, x::GenericGrayImage, ref::GenericGrayImage, peakva
     C₁, C₂ = @. (peakval * K)^2
     C₃ = C₂/2
 
-    T = promote_type(floattype(eltype(ref)), floattype(eltype(x)))
+    T = promote_type(float(eltype(ref)), float(eltype(x)))
```
Comment (johnnychen94) [inline src/ssim.jl:89]: I see the problems here, I used some latest exported function from `ColorTypes v0.8` and `FixedPointNumbers v0.6.1`. Unfortunately, not all packages in your local machine support them, so this's actually a compatibility issue.

In the meantime, `FixedpointNumbers.floattype` is used to automatically promote `N0f8` to `Float32` while still keep `AbstractFloat` not changed.

I think we could `using FixedPointNumbers`, and change `floattype` to `FixedPointNumbers.floattype`. (this function exists in FixedPointNumbers long time ago without exported)

**Review by johnnychen94 — CHANGES_REQUESTED** [review]
Hah! Thanks for the report and fix!

File: src/ssim.jl (hunk 1 above)
Comment (johnnychen94) [reply src/ssim.jl:89]: Just found out that there's a `float(x::Normed) = convert(floattype(x), x)` defined in `FixedPointNumbers`, so `float` behaves equivalently to `floattype`. Your change looks better than mine.

Comment (JeffreySarnoff) [reply src/ssim.jl:89]: lol

**Pull request merged**